src/applications/caregivers/components/PreSubmitInfo/index.jsx

```diff
@@ -81,20 +81,25 @@ const PreSubmitCheckboxGroup = ({ onSectionComplete, formData, showError }) => {
         </p>
 
         <p>
-          I certify that I am a family member of the Veteran or service member
-          named in this application or I reside with the Veteran or service
-          member or will do so upon approval.
+          I certify that I am a family member of the Veteran named in this
+          application or I reside with the Veteran, or will do so upon approval.
         </p>
 
         <p>
-          {`I agree to perform personal care services as the ${label} for the Veteran or service member named on this application.`}
+          {`I agree to perform personal care services as the ${label} for the Veteran named on this application.`}
         </p>
 
         <p>
           {`I understand that the Veteran or Veteran’s surrogate may initiate my
-            revocation as a ${label} at any time and that the VA
-            may immediately revoke this designation if I fail to comply with the
-            program requirements for continued participation in the program.`}
+            revocation as a ${label} (PCAFC) at any time. I understand that my designation 
```

Comment:
Does the label contain the spelled-out text for `PCAFC`? If it does, then I think you can reuse the abbreviation, instead of the `label`, in the paragraphs below.

Reply:
Unfortunately not PCAFC is the department and the label only has the party in this case the label is `Secondary Family Caregiver\u2019s (2)`

Reply:
Please make sure that `PCAFC `is spelled-out at the top of the page.